sovtokenfees/sovtokenfees/main.py

```diff
@@ -97,4 +97,26 @@ def filter_fees(ledger_id: int, txn: Any):
     node.master_replica.register_hook(ReplicaHooks.APPLY_PPR,
                                       three_pc_handler.check_recvd_pre_prepare)
 
+    # Pluggable request handlers initialization
+
+    # Firstly registered handlers must be txn main purposed
+    # node.write_request_manager.register_req_handler(SetFeesHandler(node.db_manager,
+    #                                                   node.write_req_validator))
+    # Replace XFER handler
+    # del node.write_request_manager.request_handlers[XFER_PUBLIC][-1]
```

Comment:
`node.write_request_manager.remove_req_handlers` should be used instead

Reply:
fixed